app/controllers/transactions_controller.rb

```diff
@@ -42,6 +44,25 @@ def destroy
     @transaction.destroy
   end
 
+  def copy
+    if @transaction.transfer?
+      @transaction = @transaction.dup
```

Comment:
@AlexanderUlitin, I checked, the fields `:created_at` and `:updated_at` are overwritten.